gen3cirrus/aws/services.py

```diff
@@ -0,0 +1,52 @@
+"""
+Amazon service for interacting with APIs
+"""
+
+import backoff
+import boto3
+
+from botocore.exceptions import ClientError
+from gen3cirrus.config import config
+from gen3cirrus.aws.utils import generatePresignedURL, generatePresignedURLRequesterPays
+
+from cdislogging import get_logger
+
+logger = get_logger(__name__, log_level="info")
+
+
+class AwsService(object):
+    """
+    Generic Amazon servicing using Boto3
+    """
+
+    def __init__(self, client):
+        self.client = client
+
+    def downloadPresignedURL(self, bucket, key, expiration):
+        """
+        Wrapper function for generating a presingned url for downloading an object
```

Comment:
```suggestion
        Wrapper function for generating a presigned URL for downloading an object
```

and other spots in this file too